src/app/features/test-records/components/base-test-record/base-test-record.component.ts

```diff
@@ -65,11 +69,23 @@ export class BaseTestRecordComponent implements AfterViewInit {
     const requiredStandardsValue = this.requiredStandards?.form.getCleanValue(this.requiredStandards?.form);
 
     latestTest = merge(latestTest, defectsValue, customDefectsValue, requiredStandardsValue, event);
+
     if (latestTest && Object.keys(latestTest).length > 0) {
       this.newTestResult.emit(latestTest as TestResultModel);
     }
   }
 
+  validateEuVehicleCategory(_event: unknown) {
```

Comment:
if I'm being super picky I'd love for the linter warnings to be handled, there's only one in dev at the moment and it's got a TODO next to it when there's a required standards model (I'm guessing). I realise most of these linter issues may just want ignoring